Traceback (most recent call last):
  File "C:\Users\dasge\AppData\Roaming\Python\Python37\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "C:\Users\dasge\AppData\Roaming\Python\Python37\site-packages\nbclient\client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\dasge\AppData\Roaming\Python\Python37\site-packages\nbclient\util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\dasge\AppData\Roaming\Python\Python37\site-packages\nbclient\util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "C:\ProgramData\Anaconda3\envs\jb\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "C:\Users\dasge\AppData\Roaming\Python\Python37\site-packages\nbclient\client.py", line 559, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "C:\Users\dasge\AppData\Roaming\Python\Python37\site-packages\nbclient\client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\dasge\AppData\Roaming\Python\Python37\site-packages\nbclient\client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
import scipy as sp
import pandas as pd
from copy import deepcopy
from pyproj import Proj, Geod
import matplotlib.pyplot as plt
from scipy.optimize import leastsq
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_10056/4282490719.py[0m in [0;36m<module>[1;34m[0m
[0;32m      3[0m [1;32mimport[0m [0mpandas[0m [1;32mas[0m [0mpd[0m[1;33m[0m[1;33m[0m[0m
[0;32m      4[0m [1;32mfrom[0m [0mcopy[0m [1;32mimport[0m [0mdeepcopy[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 5[1;33m [1;32mfrom[0m [0mpyproj[0m [1;32mimport[0m [0mProj[0m[1;33m,[0m [0mGeod[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      6[0m [1;32mimport[0m [0mmatplotlib[0m[1;33m.[0m[0mpyplot[0m [1;32mas[0m [0mplt[0m[1;33m[0m[1;33m[0m[0m
[0;32m      7[0m [1;32mfrom[0m [0mscipy[0m[1;33m.[0m[0moptimize[0m [1;32mimport[0m [0mleastsq[0m[1;33m[0m[1;33m[0m[0m

[1;31mModuleNotFoundError[0m: No module named 'pyproj'
ModuleNotFoundError: No module named 'pyproj'

